doc/classes/CodeEdit.xml

```diff
@@ -548,28 +548,28 @@
 		</member>
 		<member name="gutters_draw_breakpoints_gutter" type="bool" setter="set_draw_breakpoints_gutter" getter="is_drawing_breakpoints_gutter" default="false">
-			Sets if breakpoints should be drawn in the gutter. This gutter is shared with bookmarks and executing lines.
+			If [code]true[/code], breakpoints are drawn in the gutter. This gutter is shared with bookmarks and executing lines. Clicking the gutter will toggle the breakpoint for the line, see [method set_line_as_breakpoint].
 		</member>
 		<member name="gutters_draw_executing_lines" type="bool" setter="set_draw_executing_lines_gutter" getter="is_drawing_executing_lines_gutter" default="false">
-			Sets if executing lines should be marked in the gutter. This gutter is shared with breakpoints and bookmarks lines.
+			If [code]true[/code], executing lines are marked in the gutter. This gutter is shared with breakpoints and bookmarks. See [method set_line_as_executing].
 		</member>
 		<member name="gutters_draw_fold_gutter" type="bool" setter="set_draw_fold_gutter" getter="is_drawing_fold_gutter" default="false">
-			Sets if foldable lines icons should be drawn in the gutter.
+			If [code]true[/code], the fold gutter is drawn. In this gutter, the [theme_item can_fold_code_region] icon is drawn for each foldable line (see [method can_fold_line]) and the [theme_item folded_code_region] icon is drawn for each folded line (see [method is_line_folded]). These icons can be clicked to toggle the fold state, see [method toggle_foldable_line]. [member line_folding] must be [code]true[/code] to show icons.
 		</member>
 		<member name="gutters_draw_line_numbers" type="bool" setter="set_draw_line_numbers" getter="is_draw_line_numbers_enabled" default="false">
-			Sets if line numbers should be drawn in the gutter.
+			If [code]true[/code], the line number gutter is drawn. Line numbers start at [code]1[/code] and are incremented for each line of text. Clicking and dragging in the line number gutter will select entire lines of text.
 		</member>
 		<member name="gutters_zero_pad_line_numbers" type="bool" setter="set_line_numbers_zero_padded" getter="is_line_numbers_zero_padded" default="false">
-			Sets if line numbers drawn in the gutter are zero padded.
+			If [code]true[/code], line numbers drawn in the gutter are zero padded based on the highest number. [member gutters_draw_line_numbers] must be [code]true[/code].
 		</member>
 		<member name="indent_automatic" type="bool" setter="set_auto_indent_enabled" getter="is_auto_indent_enabled" default="false">
-			Sets whether automatic indent are enabled, this will add an extra indent if a prefix or brace is found.
+			If [code]true[/code], an extra indent will automatically be added when adding a new line if a prefix in [member indent_automatic_prefixes] is found. If a brace pair opening key is found (see [member auto_brace_completion_pairs]), the matching closing brace will be moved to another new line.
```

Comment:
I suggest moving this parenthesis for more comfortable reading.
```suggestion
			If [code]true[/code], an extra indent will automatically be added when a new line is added and a prefix in [member indent_automatic_prefixes] is found. If a brace pair opening key is found, the matching closing brace will be moved to another new line (see [member auto_brace_completion_pairs]).
```
The verb form change is not that big of a deal. It could even be wrong or have make it worse.

Reply:
Adjusted the first part to `an extra indent is automatically inserted when a new line is added and a prefix...`.